build/config.props

```diff
@@ -28,16 +28,25 @@
 
     <IsEscrowMode>false</IsEscrowMode>
 
-    <!-- Visual Studio Insertion Logic -->    
+    <!-- Visual Studio Insertion Logic -->
     <VsTargetMajorVersion>$([MSBuild]::Add(11, $(MajorNuGetVersion)))</VsTargetMajorVersion>
     <VsTargetBranch>master</VsTargetBranch>
     <VsTargetChannel>int.$(VsTargetBranch)</VsTargetChannel>
+
+    <!-- This branches are used for creating insertion PRs -->
     <VsTargetBranch Condition="'$(IsEscrowMode)' == 'true'">rel/d$(VsTargetMajorVersion).$(MinorNuGetVersion)</VsTargetBranch>
     <VsTargetChannel Condition="'$(IsEscrowMode)' == 'true'">int.d$(VsTargetMajorVersion).$(MinorNuGetVersion)</VsTargetChannel>
 
-    <!-- .NET Core SDK Insertion Logic -->    
+    <!-- .NET Core SDK Insertion Logic -->
+    <!-- when LockSDKVersion is true,  it will use the specific version in CliVersionForBuilding for building, and specific version in OverrideCliBranchForTesting for testing,  -->
+    <!-- when LockSDKVersion is false, it will ignore the properties and just use the latest version for the channel specified in CliVersionForBuilding for building, CliBranchForTesting for testing -->
+    <LockSDKVersion>true</LockSDKVersion>
+    <OverrideCliBranchForTesting Condition="'$(LockSDKVersion)' == 'false'"></OverrideCliBranchForTesting>
```

Comment:
Do we still need the locking here? 

This whole part is super confusing to me :D

Reply:
You're right. We just use the latest version now. Removed the lock part.
Thanks!